src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.ps1

```diff
@@ -218,23 +218,19 @@ function Test-VirtualNetworkCRUDWithDDoSProtection
         Assert-AreEqual "10.0.1.0/24" $expected.Subnets[0].AddressPrefix
         Assert-AreEqual true $expected.EnableDDoSProtection
         Assert-AreEqual $ddosProtectionPlan.Id $expected.DdosProtectionPlan.Id
```

Comment:
Looks like this test is skipped. Please remove the skip for this test because of this failure. 

        [Fact(Skip = "Fails with server-side error: 'An error occurred'.")]
        [Trait(Category.AcceptanceType, Category.CheckIn)]
        [Trait(Category.Owner, Category.sdnnrp)]
        public void TestVirtualNetworkCRUDWithDDoSProtection()

Reply:
Good catch, thanks. Removed